src/audio_generic.cpp

```diff
@@ -233,7 +233,7 @@ bool GenericAudio::PlayOnChannel(BgmChannel& chan, Filesystem_Stream::InputStrea
 	if (chan.decoder && chan.decoder->Open(std::move(filestream))) {
 		chan.decoder->SetPitch(pitch);
 		chan.decoder->SetFormat(output_format.frequency, output_format.format, output_format.channels);
-		chan.decoder->SetFade(0, volume, std::chrono::milliseconds(fadein));
+		chan.decoder->SetFade(volume, std::chrono::milliseconds(fadein));
```

Comment:
Is volume guaranteed to be 0 at this point?

Reply:
good find. I forgot a "SetVolume" call here.